knowledge-graph/README.md

```diff
@@ -811,20 +811,70 @@ Response body example for `Accept: application/ld+json`:
 }
 ```
 
-#### PUT /knowledge-graph/projects/:namespace/:name
+#### PATCH /knowledge-graph/projects/:namespace/:name
 
-API to update selected properies of the project with the given `namespace/name` in both the Triples Store and GitLab
+API to update project data.
 
+Each of the properties can be either set to a new value or omitted in case there's no new value for it.
+The new values should be sent as a `multipart/form-data` in case there's new image or as JSON in case no update for an image is needed.
+
+The properties that can be updated are:
+* description - possible values are:
+  * `null` for removing the current description
+  * any non-blank String value
+* image - possible values are:
+  * `null` for removing the current image
+  * any file content
```

Comment:
again embarassingly bean counting :) should we write "image file" - perhaps there are supported images and some not supported - is there also a max file size?